.buildkite/x-pack/pipeline.xpack.osquerybeat.yml

```diff
@@ -297,3 +300,22 @@ steps:
         notify:
           - github_commit_status:
               context: "x-pack/osquerybeat: Packaging Linux"
+
+      - label: ":ubuntu: x-pack/osquerybeat: Packaging Linux arm64"
+        key: "packaging-arm"
+        env:
+          PLATFORMS: "linux/arm64"
```

Comment:
now that this has been added here, we need to remove `linux/arm64` from https://github.com/elastic/beats/blob/0a3020a01da1f8038af980cb87ec1c30380e7e3f/.buildkite/x-pack/pipeline.xpack.osquerybeat.yml#L287 to ensure we stop building arm64 artifacts in the previous step that uses an amd64 worker.

Reply:
Good catch, see [`c7fdcb1` (#43019)](https://github.com/elastic/beats/pull/43019/commits/c7fdcb1235668090d14b1e9ec3d98ea66c5e01f0)